app/assets/stylesheets/screens/_papers.scss

```diff
@@ -105,6 +105,68 @@ html.control-bar-sub-nav-active #paper-body .oo-ui-toolbar {
   min-width: 100px;
 }
 
+.editor-toolbar {
+  background: rgba($tahi-white, 1.0);
+  float: none;
+  width: 1120px;
+  margin: 0 auto;
+  cursor: default;
+
+  &.disabled {
+    opacity: 0.3;
+  }
+
+  .edit-paper {
+    display: inline-block;
+    margin-left: 5px;
+  }
+
+  .separator {
+    width: 1px;
+    border-left: 1px solid #ccc;
+    margin-left: 10px;
+    margin-right: 10px;
+  }
+
+  .ve-tool-group {
+    display: inline-block;
+  }
+
+  .ve-select {
+    padding: 5px;
+    margin-left: 5px;
+    margin-right: 5px;
+    font-size: small;
+
+    .ve-option.disabled {
+      display: none;
+    }
+  }
+}
+
+.editor-toolbar .ve-tool-group .ve-tool {
```

Comment:
Merge rule `.editor-toolbar .ve-tool-group .ve-tool` with rule on line 108<br>Selector should have depth of applicability no greater than 2, but was 3